src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -207,8 +206,17 @@ private ProgressStepDefinition[] CreateWorkflowSteps(IProgressController control
             foreach (var language in languages)
             {
                 notifier.NotifyCurrentProgress(string.Format(CultureInfo.CurrentCulture, Strings.DownloadingQualityProfileProgressMessage, language.Name));
+
+                QualityProfile qualityProfileInfo;
+                if (!host.SonarQubeService.TryGetQualityProfile(this.connectionInformation, this.project, language.ServerKey, cancellationToken, out qualityProfileInfo))
+                {
+                    failed = true;
+                    break;
```

Comment:
What will _TryGetQualityProfile_ return if the particular language plugin isn't installed on the server? (there's no guarantee that either the C# or VB plugins will be installed). If the call will return false in that case then breaking out of the loop and not checking the other languages probably isn't the right thing to do.